build.rs

```diff
@@ -1,9 +1,14 @@
+#[cfg(target_arch = "x86_64")]
 fn main() {
-    let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").unwrap();
```

Comment:
You need to use this env flag, to read out the target_arch properly. This should not be deleted